src/schema/posts.ts

```diff
@@ -2344,19 +2345,22 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
       ctx: AuthContext,
       info,
     ): Promise<GQLPost> => {
-      await Promise.all([
+      // eslint-disable-next-line @typescript-eslint/no-unused-vars
+      const [post, __, ___] = await Promise.all([
```

Comment:
Should do it?

```suggestion
      const [post] = await Promise.all([
```

Reply:
I can, just need different eslint ignore then for not using all promises 😅